lychee-bin/src/formatters/stats/compact.rs

```diff
@@ -15,27 +12,6 @@ use anyhow::Result;
 
 struct CompactResponseStats(ResponseStats);
 
-// Helper function, which prints the detailed list of errors
-pub(crate) fn print_errors(stats: &ResponseStats) -> String {
-    let mut errors = HashMap::new();
-    errors.insert(
-        "HTTP",
-        stats.errors - stats.redirects - stats.timeouts - stats.unknown,
```

Comment:
That's the reason for the bug. Redirects, timeouts, and unknown link statuses aren't counted as errors, but we still substract them from the total number of errors to calculate the HTTP errors. We might also just remove the HTTP errors in the output, but the counts of all individual "errors" would not sum up to the total error count and be even more confusing to the user. Safer to remove that than show inaccurate information.